sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1290,6 +1290,7 @@ struct sof_sdw_endpoint {
 
 	u32 link_mask;
 	const char *codec_name;
+	const char *name_prefix;
```

Comment:
commit message

this facilitates ?

Reply:
lol you would think with english being my first language I would be better at it. Will fix that up.

Reply:
all the complicated words in English are really French, aren't they?